db/migrations/20161013155400_buzzerdb.sql

```diff
@@ -1,23 +1,13 @@
 -- +goose Up
-CREATE TABLE Device (
```

Comment:
I thought we might want to leave this for now just to test stuff but if you think we are good that works for me